src/helm/benchmark/window_services/window_service_factory.py

```diff
@@ -82,7 +82,7 @@ def get_window_service(model_name: str, service: TokenizerService) -> WindowServ
                 window_service = GPT4WindowService(service)
             elif model_name in get_model_names_with_tag(GPT4_32K_CONTEXT_WINDOW_TAG):
                 window_service = GPT432KWindowService(service)
-            if model_name in get_model_names_with_tag(GPT_TURBO_CONTEXT_WINDOW_TAG):
+            elif model_name in get_model_names_with_tag(GPT_TURBO_CONTEXT_WINDOW_TAG):
```

Comment:
This error was causing GPT-4 to incorrectly use the GPT-2 window service and tokenizer.